app/routes/index.js

```diff
@@ -284,4 +310,71 @@ module.exports = function(app, fs) {
 			else
 				res.redirect('/admin');
 		});
+	app.post('/adduser', upload.single('profile_pic'), async function(req,res){
+		try {
+			const user = new User(req.body);
+			console.log(user)
+			if(req.file){
+				user.profile_pic = req.file.tempFilePath;
+			}
+			await user.save()
+			res.json("User added successfully");
+		} catch (error) {
+			console.log("error");
+			res.json(error);
+			
+		}
+		
+	});
+	app.route('/addevent').
+	post(adminAuth, async function(req,res){
+		try {
+			const event = new Event(req.body);
+			event.save()
+			res.json(event);
+		} catch (error) {
+			console.log("error");
+			res.json(error);
+			
+		}
+	})
+	app.route(adminAuth, '/addteamhere').
+	post(async function(req,res){
+		try {
+			const yearobj = await Team.findOne({ year: req.body.year});
+			const curr_user=await User.findOne({name : req.body.name});
+			if(curr_user==null)
+			{
+				res.json("current user not registered: error")
```

Comment:
Please send a proper json format when using `res.json`. You could maintain a fixed schema while returning. For example to send an error response:
```
res.json(
{
err: true,
msg: "error message here"
}
)
```
To send an ok response - 
```
res.json({err: false, msg: ""})
```

Reply:
Done